.travis.yml

```diff
@@ -0,0 +1,35 @@
+---
+sudo: required
+
+env:
+  - distribution: ubuntu
+    version: xenial
+    ansible: 2.4.3.0
+  - distribution: ubuntu
+    version: xenial
+    ansible: 2.3.1.0
+  - distribution: ubuntu
+    version: trusty
+    ansible: 2.4.3.0
+  - distribution: ubuntu
+    version: trusty
+    ansible: 2.3.1.0
+    
+services:
+  - docker
+
+before_install:
+  - 'sudo docker pull ${distribution}:${version}'
+  - 'sudo docker build --no-cache --rm --file=travis/Dockerfile.${distribution}-${version} --build-arg ansible_version=${ansible} --tag=${distribution}-${version}-${ansible}:ansible travis'
+
+script:
+  - container_id=$(mktemp)
```

Comment:
This looks like a dangerous method of doing IDs. I hope that Travis can help.

Reply:
What's dangerous about it?